test/fixtures/output/network/list.golden

```diff
@@ -0,0 +1,6 @@
+     ID    | Environment |         Name         | Cloud |   Region    |     CIDR      |             Zones              | DNS Resolution | Phase |   Active Connection Types    
```

Comment:
I would recommend removing environment from this list, since the user already knows which environment they're in. Running `confluent environment use` or passing `--environment` is a prerequisite to running this command.

Reply:
You might want to consider other ways to shorten this output, too! Right now this requires having a pretty wide screen to see the output.

Reply:
@ejagade @kabraham13 do you guys have any opinions on this? Perhaps we can remove `Environment` and `DNS Resolution` to shorten this output, is there any other field that we can remove?

Reply:
I am fine removing `Environment` we need to keep `DNS Resolution` though

Reply:
yeah we could lose `Environment` and `DNS Resolution` for list ouput. the user can do a `get` to get a detailed output.
we can remove the `cidr` and maybe `zones` too? @kabraham13

Reply:
I take it back :-) let's go with Kevin's proposal

Reply:
I will remove `Environment` and keep the rest.

Reply:
Also, circling back to the [discussion](https://github.com/confluentinc/cli/pull/2193#discussion_r1295132357) while implementing the `confluent network describe` command

Do we want to add the following to the `describe` command as @ejagade suggested?
```
network.Status.GetCloud().NetworkingV1AwsNetwork
network.Status.GetCloud().NetworkingV1AzureNetwork
network.Status.GetCloud().NetworkingV1GcpNetwork
```

Perhaps we can add the required fields from the response
For `AzureNetwork`, add `vnet` and `subscription`
For `AwsNetwork`, add `vpc` and `account`
For `GcpNetwork`, add `project` and `vpc_network`

What do you guys think?

Reply:
^ @ejagade @kabraham13 any concern/opinion if I add the fields above? If not, I will add them in another PR.

Reply:
No, please go ahead and add them

Reply:
>Do we want to add the following to the describe command as @ejagade suggested?
@syoung-cflt you can add them as a separate PR

Reply:
Yeah I will create another ticket and add them as a separate PR.